### PR TITLE
Saving slinks into db

### PR DESCRIPTION
**Pull request by junyuanxue**

### CODE REVIEW

File: bin/www [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+/**
+ * Module dependencies.
+ */
+
+var app = require('../server');
+var http = require('http');
+var models = require("../models");
+
+/**
+ * Get port from environment and store in Express.
```
Comment (JojSh) [inline bin/www]: Some comments to remove.

**Comment by yasgreen93** [comment]
:shipit:

File: public/main.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+body {
+  background-color: #2E2E2E;
+  color: #fff;
+  font-family: 'Karla', sans-serif;
+  margin: 0 auto;
+  max-width: 1280px;
+  padding: 0;
+}
+
+#practice-text {
```
Comment (JojSh) [inline public/main.css:15]: We can remove this now.

File: public/main.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  color: yellow;
+  font-size: 30pt;
+  vertical-align: middle;
+}
+
+#heading {
+
```
Comment (JojSh) [inline public/main.css:22]: And this.

File: public/stylesheet/main.css [hunk 4]
```diff
@@ -0,0 +1,23 @@
+html {
```
Comment (JojSh) [inline public/stylesheet/main.css:1]: duplicate CSS files? One needs to be deleted.

**Pull request merged**